lib/elements/multiselect.js

```diff
@@ -145,12 +146,12 @@ class MultiselectPrompt extends Prompt {
   }
 
   renderInstructions() {
-    return `
-Instructions:
+    return this.instructions ? `
+Instructions: ${this.instructions}
```

Comment:
I haven't tested it but I think if `instructions` is set to `true` it will print _true_.

I believe the best would be something like...
- if `typeof instructions === 'string'` and it is not empty, use it as the instruction text
- otherwise if `instructions` option evaluates to true, print the default instructions
- and if `instructions` evaluates to false, do not show any instructions

```suggestion:-2+4
return this.instructions && typeof this.instructions === 'string' ? this.instructions
	: this.instructions ? 'DEFAULT INSTRUCTIONS'
	: '';
```